Dockerfile

```diff
@@ -1,12 +1,21 @@
+# NOTE: Since the `:latest` tag can have npm version changes, we are using
+#       a specific version tag. Container build errors have come up locally
+#       and via github action workflow when `:latest` is updated.
+#
+# Image info: https://catalog.redhat.com/software/containers/ubi9/nodejs-18/62e8e7ed22d1d3c2dfe2ca01
+# Relevant PRs:
+#   - https://github.com/konveyor/tackle2-ui/pull/1746
+#   - https://github.com/konveyor/tackle2-ui/pull/1781
+
 # Builder image
-FROM registry.access.redhat.com/ubi9/nodejs-18:latest as builder
+FROM registry.access.redhat.com/ubi9/nodejs-18:1-88 as builder
 
 USER 1001
 COPY --chown=1001 . .
-RUN npm clean-install && npm run build && npm run dist
+RUN npm clean-install --ignore-scripts && npm run build && npm run dist
 
 # Runner image
-FROM registry.access.redhat.com/ubi9/nodejs-18-minimal:latest
+FROM registry.access.redhat.com/ubi9/nodejs-18-minimal:1-93
```

Comment:
the most current version fixes some security issues that were not fixed in 1-93 and base 1-103 
https://catalog.redhat.com/software/containers/ubi9/nodejs-18-minimal/62e8e919d4f57d92a9dee838?architecture=amd64&image=6601e09c5781451e968f8dd5

Reply:
It would probably be ok to have the run container use `:latest` since things have already been built to the dist folder and we're just running it.